doc/overview.md

```diff
@@ -21,8 +21,7 @@ Comment Here
 ---
 
 ![bg 70%](https://images.squarespace-cdn.com/content/v1/606d378755a86f589aa297b7/1653397531343-6M4IQ4JWDQV1SQ8W17UN/HumbleHawksbill_TransparentBG-NoROS.png)
-![bg 70%](https://images.squarespace-cdn.com/content/v1/606d378755a86f589aa297b7/1684793822030-O5GLQ81TY0M8RZ7I422Q/IronIrwiniLogo.jpg?format=750w)
-![bg 70%](https://global.discourse-cdn.com/business7/uploads/ros/optimized/3X/a/e/ae61107095e154a1606d4ea7080065fe0bee52af_2_548x499.png)
+![bg 70%](https://images.squarespace-cdn.com/content/v1/606d378755a86f589aa297b7/ebf9b1d5-45b7-4a73-8f48-dc5d3f4fc8fc/JazzyJalisco_Final.png?format=1500w)
```

Comment:
for some reason, this cannot be fetched and displayed in pdf or html format when it is exported.